lib/devise/jwt.rb

```diff
@@ -63,6 +62,9 @@ module JWT
 
     def self.forward_to_warden(setting, value)
       Warden::JWTAuth.config.send("#{setting}=", value)
+      Warden::JWTAuth.config.send("#{setting}") # Evaluate the setting to remove the proc
     end
   end
 end
+
+require 'devise/jwt/railtie'
```

Comment:
Why is it needed to load it after the configuration and not before with the rest of dependencies?

Reply:
On runtime, it dont matter. But when you debug railties, i prefer to have the whole library loaded first. 
Before doing that, the .config was not defined yet.
I can revert if you want.